Framework/PythonInterface/mantid/api/src/Exports/DetectorInfo.cpp

```diff
@@ -0,0 +1,14 @@
+#include "MantidAPI/DetectorInfo.h"
+#include <boost/python/class.hpp>
+
+using Mantid::API::DetectorInfo;
+using namespace boost::python;
+
+void export_DetectorInfo() {
+  class_<DetectorInfo, boost::noncopyable>("DetectorInfo", no_init)
+      .def("size", &DetectorInfo::size, (arg("self")),
```

Comment:
@SimonHeybrock You're example would have been my first thought too but I agree that most code does not need to iterate over the detectors at this level. It may be better to wait until we can find a good example of a more real use case for this.